diff --git a/node_modules/pixi-viewport/@types/index.d.ts b/node_modules/pixi-viewport/@types/index.d.ts
index 858b377..bba75aa 100644
--- a/node_modules/pixi-viewport/@types/index.d.ts
+++ b/node_modules/pixi-viewport/@types/index.d.ts
@@ -1,4 +1,4 @@
-import * as PIXI from 'pixi.js'
+
 
 type DirectionType = 'all' | 'x' | 'y'
 type UnderflowType = 'center' | 'top' | 'left' | 'right' | 'bottom' | (string & {})
